actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -23,7 +23,8 @@ class ExceptionWrapper
       "ActionView::MissingTemplate"         => "missing_template",
       "ActionController::RoutingError"      => "routing_error",
       "AbstractController::ActionNotFound"  => "unknown_action",
-      "ActionView::Template::Error"         => "template_error"
+      "ActionView::Template::Error"         => "template_error",
+      "ActionController::MissingExactTemplate" => "missing_exact_template"
```

Comment:
Let's add a trailing comma, so this line doesn't have to change next time we add an entry.

Also, I think we need to add our new exception to `rescue_responses` above as well (pointing to `:not_acceptable`, same as the existing UnknownFormat)